### PR TITLE
[ARM plugin] Fix failed tests

### PR DESCRIPTION
**Pull request by allnes**
https://github.com/openvinotoolkit/openvino/pull/13149

### CODE REVIEW

File: modules/arm_plugin/src/arm_config.cpp [hunk 1]
```diff
@@ -61,8 +65,12 @@ InferenceEngine::Parameter Configuration::Get(const std::string& name) const {
         return {_ref};
     } else if (name == CONFIG_KEY_INTERNAL(LP_TRANSFORMS_MODE)) {
         return {_lpt};
+    } else if (name == CONFIG_KEY(ENFORCE_BF16)) {
+        return {_enforceBF16};
```
Comment (ilya-lavrenov) [inline modules/arm_plugin/src/arm_config.cpp:69]: does ARM plugin really support this option? I suppose it comes from Intel CPU plugin..

Comment (allnes) [reply modules/arm_plugin/src/arm_config.cpp:69]: ARM plugin doesn't support, but without this flag, behavior tests are failed, and same PERORMANCE_HINT

Comment (ilya-lavrenov) [reply modules/arm_plugin/src/arm_config.cpp:69]: maybe this is the problem of behavior tests, but not of the plugin?

Comment (allnes) [reply modules/arm_plugin/src/arm_config.cpp:69]: Our behavior tests don't use `ENFORCE_BF16` and `PERORMANCE_HINT` keys, and I don't know reasons but I can hardcode this keys (turn off ENFORCE_BF16) and set latency as performance hint

Comment (ilya-lavrenov) [reply modules/arm_plugin/src/arm_config.cpp:69]: but user will except that he can affect plugin behavior with such keys, but they don't mean anything for ARM..
it's a confusion for users

File: modules/arm_plugin/src/arm_converter/arm_converter_nms.cpp [hunk 2]
```diff
@@ -98,7 +98,7 @@ static void nms5(const float* boxes_data,
                                                     sort_result_descending);
 
     auto max_valid_outputs = out_shape[0];
-    ngraph::runtime::reference::nms5_postprocessing(outputs,
+    ngraph::runtime::reference::nms_postprocessing(outputs,
```
Comment (ilya-lavrenov) [inline modules/arm_plugin/src/arm_converter/arm_converter_nms.cpp:101]: need to finally find out what's wrong with nms5

File: modules/arm_plugin/src/arm_config.hpp [hunk 3]
```diff
@@ -37,9 +37,11 @@ struct Configuration {
 
     bool _exclusiveAsyncRequests = false;
     bool _perfCount              = true;
+    bool _enforceBF16            = false;
     bool _ref                    = true;
     bool _lpt                    = true;
     bool _dump                   = false;
+    ov::hint::PerformanceMode _perfHint = ov::hint::PerformanceMode::UNDEFINED;
```
[the remark this answers is not in the document]
Comment (ilya-lavrenov) [reply modules/arm_plugin/src/arm_config.hpp:44]: let's try to propagate this value to stream executor to have:
1. single stream with all big cores for latency case
2. AUTO streams in tput mode

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**